randovania/games/samus_returns/logic_database/Area 4 Central Caves.txt

```diff
@@ -696,12 +696,17 @@ Extra - asset_id: collision_camera_013
   * Pickup 85; Category? Minor
```

Comment:
What about area 8:
- nest nodule/nest network


Also queen arena should be a blocked open passage

Reply:
There's no open passages in a8 except for after queen

Reply:
Database disagrees

Reply:
Wow I completely forgot about that one

Reply:
TBF, its logical implications in vanilla are completely useless.
It would enable tho for the bottom door to be a permanently locked door :eyes: